app/grants/models/grant.py

```diff
@@ -488,7 +488,7 @@ class Meta:
     )
 
     in_active_clrs = models.ManyToManyField(
-        GrantCLR,
+        "GrantCLR",
```

Comment:
Should this be a string?

Reply:
@thelostone-mc In this case, yes. I've been doing that to prevent circular imports.